CRM/OptionValueCH/CreateAPIWrapper.php

```diff
@@ -26,7 +26,10 @@ public function fromApiInput($apiRequest) {
           'option_group_id' => is_int($optionGroupID) ? $optionGroupID : civicrm_api3('OptionGroup', 'getvalue', ['name' => 'ch_fund', 'return' => 'id']),
           'return' => 'id',
         ]);
-        CRM_Core_DAO::executeQuery(sprintf("UPDATE civicrm_option_value SET `value` = '%s' WHERE id = %d", $apiRequest['params']['value'], $id));
+        CRM_Core_BAO_OptionValue::create([
```

Comment:
Is this just using a better coding pattern, or is it intended to also change behaviour?

Reply:
It about the former part, to improve the code pattern by relying on core function to update the optionValue value rather then sql query. In addition, this core function also updates the custom field's option value in [here](https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/BAO/OptionValue.php#L171). I have captured this behavior in this [new UT](https://github.com/CanadaHelps/chfunds/pull/18/files#diff-928950ff08b305a1a29be653fc5d032fR211) which confirms that on updating an OptionValue value also update the associated contribution's custom field value.